Pipfile

```diff
@@ -23,9 +23,10 @@ pytest = "*"
 flask-pytest = "*"
 pytest-invenio = "*"
 python-dotenv = "*"
+setuptools = "~=57.5.0"
```

Comment:
Specifying the `setuptools` version in the _Pipfile_ does not solve it 😕

_Pipfile_ is designed to contain project dependencies. In this case, `setuptools` is not a project dependency _per se_, **but a tool used by certain packages during the building / installation stage**.

I believe there is no workaround to avoid:
```
pipenv run python3.8 -m pip install setuptools '~=57.5.0'
pipenv install --dev
```

Could you write that in the docs?

Reply:
I will add this to documentation before merging the PR